testcontainers/arangodb.py

```diff
@@ -0,0 +1,72 @@
+"""
+ArangoDB container support.
+"""
+from os import environ
+from testcontainers.core.config import MAX_TRIES
+from testcontainers.core.generic import DbContainer
+from testcontainers.core.waiting_utils import wait_for_logs
+
+
+class ArangoDbContainer(DbContainer):
+    """
+    ArangoDB container.
+
+    Example
+    -------
+    The example will spin up a ArangoDB container.
+    You may use the :code:`get_connection_url()` method which returns a arangoclient-compatible url
+    in format :code:`scheme://host:port`. As of now, only a single host is supported (over HTTP).
+    ::
+
+        with ArangoContainer("arangodb:3.9.1") as arango:
+            client = ArangoClient(hosts=arango.get_connection_url())
+
+            # Connect
+            sys_db = arango_client.db(username='root', password='')
+
+            # Create a new database named "test".
+            sys_db.create_database("test")
+    """
+    def __init__(self,
+                 image="arangodb:latest",
+                 port_to_expose=8529,
+                 hostname='localhost',
+                 arango_root_password='passwd',
+                 arango_no_auth=None,
```

Comment:
For `arango_no_auth` and `arango_random_root_password`, would it be possible to turn them into booleans (defaulting to `False`) to make the interface "more pythonic" (I appreciate it's not a very well defined concept)? Then the user doesn't have to know that passing a string of `1` will have the desired effect, but they can rely on the python semantics of `True` or `False`. What do you think?

Reply:
I think that by definition of falsy values also includes None (and empty collections, etc.) - I'm fine with aligning to your request - However I think even what we did here (None) is tricky concept and should be default to actual string of "0". 
for the following reasons:
1. aligning to ArangoDB documentation 
    https://www.arangodb.com/docs/stable/deployment-single-instance-manual-start.html

2. since it env var. we can't do os.environ['KEY'] = False/None (is it do expect string values) - I do understand that it not the same as it eventually passing to Docker through -e , but we do need to remember it originated in env variable . because we wrap the Docker python-sdk, it might be OK (they take care the convert of the Falsy env params)

TLDR;
- Resolved. however, I do have some concerns. (above)
- Removed additional input (hostname) -> not relevant as of now and can be "calculated" from matched internal func